src/trampoline.js

```diff
@@ -14,34 +16,33 @@ import isFunction from './isFunction';
  * @return {*} The result that is not a function
  * @example
  *
- * function evenOline(n) {
- *   if (n === 0)
- *     return true;
- *   else
- *     return R.partial(oddOline, [Math.abs(n) - 1]);
- * }
+ * const evenOline = (n) => {
+ *  if (n === 0) {
+ *   return true;
+ *  }
+ *  return R.partial(oddOline, [Math.abs(n) - 1]);
+ * };
  *
- * function oddOline(n) {
- *   if (n === 0)
- *     return false;
- *   else
- *     return R.partial(evenOline, [Math.abs(n) - 1]);
- * }
+ * const oddOline = (n) => {
+ *  if (n === 0) {
+ *    return false;
+ *  }
+ *  return R.partial(evenOline, [Math.abs(n) - 1]);
+ * };
  *
  * RA.trampoline(oddOline, 3); //=> true
  * RA.trampoline(evenOline, 200000); //=> true
  * RA.trampoline(oddOline, 300000); //=> false
  * RA.trampoline(evenOline, 200000000); //=> true
  */
-
-const trampoline = (func, ...args) => {
+const trampoline = curryN(2, (func, ...args) => {
```

Comment:
This notation is never used in Ramda or Ramda Adjunct as it is confusing. Function signature should be total and arity should be ported always correctly. In order to achieve these properties the signature have to look like this:
```suggestion
const trampoline = curryN(2, (func, args) => {
```
Usage:
```js
RA.trampoline(Math.max, [1, 2, 3]); // => 3